### PR TITLE
Add 1.2.0 version of the WildFly Bootable Jar Stack

### PR DESCRIPTION
**Pull request by RomanNikitenko**
### What does this PR do?:

The order of components in a devfile matters to the Che-Code editor - the first one in the list is used for running the editor.
The first component for the `WildFly Bootable Jar` Stack is `jaeger`, it's based on the `quay.io/jaegertracing/all-in-one:1.52` image.
At the moment it's not possible to start the editor in the `quay.io/jaegertracing/all-in-one:1.52` container due to missing libraries required by Node.js.
The problem can be fixed by simply rearranging components in the devfile.
So, the devfile has the same set of components/images, but the order is changed.

I provided my changes as `1.2.0` version of the devfile.

### Which issue(s) this PR fixes:

https://github.com/eclipse/che/issues/22525
https://github.com/eclipse/che/issues/22787

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

File: stacks/java-wildfly-bootable-jar/1.2.0/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+      checkoutFrom:
+        remote: wildfly-quickstart
+        revision: 26.1.0.Final
+      remotes:
+        wildfly-quickstart: https://github.com/wildfly/quickstart.git
+    subDir: microprofile-rest-client
+components:
+  - name: wildfly
+    container:
+      image: registry.access.redhat.com/ubi8/openjdk-11
```
Comment (thepetk) [inline stacks/java-wildfly-bootable-jar/1.2.0/devfile.yaml:95]:
```suggestion
      image: registry.access.redhat.com/ubi8/openjdk-11:1.18-2
```

**Review by thepetk — CHANGES_REQUESTED** [review]

**Comment by thepetk** [comment]
> @thepetk could you please take another look?

@ibuziuk @RomanNikitenko from my side is ok. It might be a very small change but in general I like that we are introducing a new version instead of updating the existing one.

My proposed change was only to activate the renovate bot for this image, cause without a tag we are not able to automatically update the dependency.

@ehsavoie as you are listed as an owner of this stack could you also review the PR, so we are able to merge it?

**Review by thepetk — APPROVED** [review]
/lgtm

Needs to be approved by the stack owners too :)

**Comment by michael-valdron** [comment]
> /lgtm
> 
> Needs to be approved by the stack owners too :)

cc: @ehsavoie

**Review by ibuziuk — APPROVED** [review]

**Review by ehsavoie — COMMENTED** [review]
LGTM

**Pull request merged**